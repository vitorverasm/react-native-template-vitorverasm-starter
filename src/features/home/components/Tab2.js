import React from 'react';
import { Text, View } from 'react-native';

const Tab2 = ({ props }) => (
  <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
    <Text>TAB 2</Text>
  </View>
);

export default Tab2;
