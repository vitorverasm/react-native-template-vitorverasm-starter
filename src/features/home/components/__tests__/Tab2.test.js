import { shallow } from 'enzyme';
import React from 'react';
import Tab2 from '../Tab2';

describe('Tab2.js', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Tab2 />);
    expect(wrapper).toMatchSnapshot();
  });
});
