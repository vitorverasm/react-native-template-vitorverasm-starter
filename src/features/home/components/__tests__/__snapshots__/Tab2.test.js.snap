// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tab2.js renders correctly 1`] = `ShallowWrapper {}`;
